README.md

```diff
@@ -31,6 +31,10 @@ Build from source using [go install](https://golang.org/ref/mod#go-install):
 
 > Note: Go 1.18 or greater is currently required.
 
+> Note: On Windows, first, make sure that you have Go and GCC installed:
+> * Go: https://go.dev/doc/install
+> * GCC: https://jmeubank.github.io/tdm-gcc/ (pick an install for both, 32 & 64 bits)
```

Comment:
This is true for non-windows users as well. We could probably lift some of the language from here: https://github.com/mattn/go-sqlite3#installation

Reply:
Thanks for the pointer, @jlegrone. 

All the info needed to start TemporalLite is, indeed, included in the file you are pointing too. It's a lot of details. :)

To me it was not helpful when I was dealing with it initially, because the file reads in a way that I thought it was meant for people who want to build TemporalLite. On contrary, I was just looking for the quickest way to get it running and using it for workflows.

I installed Go, but `go install github.com/DataDog/temporalite/cmd/temporalite@latest` failed because I did not have gcc. The issues is that getting gcc is _very_ OS specific:

On Linux it's probably already there, or you just use your favorite package manager.

On Windows I (and many people) use the Microsoft tool set. GCC is not common to be there already. And it long time to get what I needed:
I googled for GCC for Windows and found a ton of noise. Then, I installed something that "looked reasonable", but it later turned out to be an ancient, long outdated version. Then, I found another download location and installed _that_. The version was fine, but the ``go install xxx` failed with a cryptic error. So, another debugging / Googling session was required. It then turned out that the MinGW distribution I installed did not include the right bitt-ness for gcc. So, I needed to remove it and keep looking for another distribution. Eventually I found TDM-GCC and it worked. :)

Basically, after all that I decided to make this PR to save other people from having to go through it. :)

So, in a nutshell, it is not same across OSes. On Linux (please correct me if I am wrong) you need to use your package manager, not a direct download, and I am not sure what you need to do on MacOS.

(Same for Go, I presume)

In the top-level readme we want to be very brief. The following should cover Win and Linux, but please let me know what do add for MacOS (I do not use it). Once I hear from you, I can update the PR.

 ---

> Note: First, make sure that you have Go and GCC installed:
> * Windows:
>   - Go: https://go.dev/doc/install
>   - GCC: https://jmeubank.github.io/tdm-gcc/ (pick an install for both, 32 & 64 bits)
> * MacOS: _please advise what to put here_
> * Linux: Use your distribution's package manager.

 ---

```md
> Note: First, make sure that you have Go and GCC installed:
> * Windows:
>   - Go: https://go.dev/doc/install
>   - GCC: https://jmeubank.github.io/tdm-gcc/ (pick an install for both, 32 & 64 bits)
> * MacOS: _please advise what to put here_
> * Linux: Use your distribution's package manager.
```

 ---